File: Toggly.FeatureManagement.NET/Toggly.FeatureManagement/Data/SignedDefinitionsResponse.cs
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Toggly.FeatureManagement.Data
{
    /// <summary>
    /// Signed definitions response
    /// </summary>
    public class SignedDefinitionsResponse
    {
        /// <summary>
        /// List of feature definitions
        /// </summary>
        [JsonPropertyName("defs")]
        public List<FeatureDefinitionModel>? Defs { get; set; }

        /// <summary>
        /// Signature of the definitions
        /// </summary>
        [JsonPropertyName("signature")]
        public string? Signature { get; set; }

        /// <summary>
        /// Timestamp of the definitions
        /// </summary>
        [JsonPropertyName("timestamp")]
        public long Timestamp { get; set; }

        /// <summary>
        /// Key Id of the signature
        /// </summary>
        [JsonPropertyName("kid")]
        public string? Kid { get; set; }
    }
}

File: Toggly.FeatureManagement.PHP/src/Toggly/FeatureManagement/Models/SignedDefinitionsResponse.php
<?php

namespace Toggly\FeatureManagement\Models;

/**
 * Signed definitions response
 */
class SignedDefinitionsResponse
{
    /**
     * List of feature definitions
     * @var FeatureDefinition[]
     */
    public array $defs = [];

    /**
     * Signature of the definitions
     */
    public string $signature;

    /**
     * Timestamp of the definitions
     */
    public int $timestamp;

    /**
     * Key ID of the signature
     */
    public string $kid;

    public function __construct(array $data = [])
    {
        if (isset($data['defs'])) {
            $this->defs = array_map(function ($def) {
                return $def instanceof FeatureDefinition ? $def : new FeatureDefinition($def);
            }, $data['defs']);
        }
        if (isset($data['signature'])) {
            $this->signature = $data['signature'];
        }
        if (isset($data['timestamp'])) {
            $this->timestamp = (int)$data['timestamp'];
        }
        if (isset($data['kid'])) {
            $this->kid = $data['kid'];
        }
    }
}
